### DISCUSSION TITLE
i dont have idea why this not work?

### DISCUSSION THREAD

**Discussion by radthenone** [usebruno/bruno#2568] [General]
![image](https://github.com/usebruno/bruno/assets/120286486/ba10bcf9-2bfc-4857-a60a-3a56ac7e1931)
then
![image](https://github.com/usebruno/bruno/assets/120286486/15124ee8-4a6e-467a-aa46-a73a4dca2031)
and i want this to use
![image](https://github.com/usebruno/bruno/assets/120286486/8b543f78-647c-42ab-9684-f06e1dbee0fc)
and my program see this
![image](https://github.com/usebruno/bruno/assets/120286486/f37568ed-34fe-44e0-ba12-d2b4af8255f1)

**Comment by Its-treason** [comment]
It's a Bug, variables in Multipart form are not replaced: https://github.com/usebruno/bruno/issues/352

**Reply by radthenone** [reply]
Omg thanks, I spent 2 hours wondering what I was doing wrong. But this program is useless XD

**Reply by fgrande** [reply]
"Useless" ? Really ?
